From 2a59809091c2a0961fb4d044b6ab81aa6f415cb5 Mon Sep 17 00:00:00 2001
From: Tang Yuantian <andy.tang@nxp.com>
Date: Thu, 10 Jan 2019 05:52:05 -0500
Subject: [PATCH 434/706] GPU: CL192825 Disable RA and SE clock gating.

Workaround for hang issue on some cases.
Impact: power consumption will increase about 10% for GPU module.

Signed-off-by: Qiang Fu <qiang.fu@vivantecorp.com>
Signed-off-by: Andy Tang <andy.tang@nxp.com>
(cherry picked from commit bb2a92352a9f00ce36c30430bd550b1636094544)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/mxc/hal/kernel/arch/gc_hal_kernel_hardware.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/mxc/hal/kernel/arch/gc_hal_kernel_hardware.c b/drivers/mxc/hal/kernel/arch/gc_hal_kernel_hardware.c
index 9a2fd6d95b1e..9c215c96e7a7 100644
--- a/drivers/mxc/hal/kernel/arch/gc_hal_kernel_hardware.c
+++ b/drivers/mxc/hal/kernel/arch/gc_hal_kernel_hardware.c
@@ -460,7 +460,7 @@ _GetHardwareSignature(
 /* Set to 1 to enable module clock gating debug function.
 *  Following options take effect when it is set to 1.
 */
-#define gcdDEBUG_MODULE_CLOCK_GATING          0
+#define gcdDEBUG_MODULE_CLOCK_GATING          1
 /* Set to 1 to disable module clock gating of all modules. */
 #define gcdDISABLE_MODULE_CLOCK_GATING        0
 /* Set to 1 to disable module clock gating of each module. */
@@ -469,8 +469,8 @@ _GetHardwareSignature(
 #define gcdDISABLE_PE_CLOCK_GATING            0
 #define gcdDISABLE_SH_CLOCK_GATING            0
 #define gcdDISABLE_PA_CLOCK_GATING            0
-#define gcdDISABLE_SE_CLOCK_GATING            0
-#define gcdDISABLE_RA_CLOCK_GATING            0
+#define gcdDISABLE_SE_CLOCK_GATING            1
+#define gcdDISABLE_RA_CLOCK_GATING            1
 #define gcdDISABLE_RA_EZ_CLOCK_GATING         0
 #define gcdDISABLE_RA_HZ_CLOCK_GATING         0
 #define gcdDISABLE_TX_CLOCK_GATING            0
-- 
2.17.1

